prometheusbp/metrics_buckets_old.go

```diff
@@ -0,0 +1,14 @@
+//go:build !metrics_spec
+// +build !metrics_spec
+
+package prometheusbp
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// DefaultBuckets is the default bucket values for a prometheus histogram metric.
+var DefaultBuckets = prometheus.ExponentialBuckets(0.0001, 2.5, 14) // 100us ~ 14.9s
+
+// KafkaBuckets is the set of bucket values for Kafka-related prometheus histogram metrics.
+var KafkaBuckets = prometheus.ExponentialBucketsRange(1e-4, 10, 10) // 100us - 10s
```

Comment:
```suggestion
var (
	// DefaultBuckets is the default bucket values for a prometheus histogram metric.
	DefaultBuckets = prometheus.ExponentialBuckets(0.0001, 2.5, 14) // 100us ~ 14.9s
	// KafkaBuckets is the set of bucket values for Kafka-related prometheus histogram metrics.
	KafkaBuckets = prometheus.ExponentialBucketsRange(1e-4, 10, 10) // 100us - 10s
)
```
If I am not mistaken we follow the group similar values style guideline here: https://github.com/uber-go/guide/blob/master/style.md#group-similar-declarations